app/components/newsletter/newsletter-header-overview.js

```diff
@@ -181,6 +190,41 @@ export default class NewsletterHeaderOverviewComponent extends Component {
     this.showConfirmPublishAll = false;
   }
 
+  async canSendMailCampaign() {
+    const agenda = await this.store.queryOne('agenda', {
+      'filter[created-for][:id:]': this.args.meeting.id,
+      sort: '-created', // serialnumber
+    });
+    const themisPublicationActivities = await this.store.queryAll('themis-publication-activity', {
+      filter: {
+        meeting: {
+          ':id:': this.args.meeting.id,
+        },
+      },
+    });
+    const themisPublicationActivity = themisPublicationActivities.find((activity) => activity.scope.includes(CONSTANTS.THEMIS_PUBLICATION_SCOPES.DOCUMENTS));
+
+    const hasDocumentPublicationPlanned = isPresent(themisPublicationActivity?.plannedDate);
+    const hasThemesOrNotas = (await this.store.count('agendaitem', {
+      filter: {
+        agenda: {
+          ':id:': agenda.id,
+        },
+        type: {
+          ':uri:': CONSTANTS.AGENDA_ITEM_TYPES.NOTA,
+        },
+        treatment: {
+          'newsletter-info': {
+            ':has:themes': true,
+          },
+        },
+      },
+      'page[size]': PAGE_SIZE.AGENDAITEMS,
```

Comment:
Page size is not needed for `this.store.count()`

Reply:
I removed this now.